### PR TITLE
Fix/1914 better simple search

### PR DESCRIPTION
**Pull request by inkhey**
related to #1914.
this add some tests for elasticsearch too (similar as thoses added to simple search)

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: backend/tracim_backend/tests/functional/test_elasticsearch_search.py [hunk 1]
```diff
@@ -138,6 +138,144 @@ def test_api___elasticsearch_search_ok__by_filename(
...
 
+    @parameterized.expand(
+        [
+            # content_name, search_string, nb_content_result, first_content_name, content_body
+            # exact syntax
+            ("good practices", "texttosearch", 1, "good practices", "texttosearch")
+        ]
+    )
+    def test_api___elasticsearch_search_ok__by_description(
+        self, content_name, search_string, nb_content_result, first_content_name, content_body
```
Comment (lebouquetin) [inline backend/tracim_backend/tests/functional/test_elasticsearch_search.py]: as discussed, maybe rename:
- rename content_name to created_content_name
- rename first_content_name to first_result_content_nale
- move content_body to second position and rename it to created_content_body
- for search_string, use a subpart of the content_body in order to make the test case more clear. Eg content_body is "I have fixed a bug" and search string is "bug"

File: backend/tracim_backend/tests/functional/test_elasticsearch_search.py [hunk 2]
```diff
@@ -138,6 +138,144 @@ def test_api___elasticsearch_search_ok__by_filename(
...
+                1,
+                "good practices",
+                "texttosearch",
+                "another comment texttosearch",
+            ),
+        ]
+    )
+    def test_api___elasticsearch_search_ok__by_comment_content(
+        self,
+        content_name,
```
Comment (lebouquetin) [inline backend/tracim_backend/tests/functional/test_elasticsearch_search.py]: same remarks about naming and order of parameters

File: backend/tracim_backend/tests/functional/test_simple_search.py [hunk 3]
```diff
@@ -160,6 +160,135 @@ def test_api___simple_search_ok__by_filename(
...
+    @parameterized.expand(
+        [
+            # content_name, search_string, nb_content_result, first_content_name, content_body
+            # exact syntax
+            ("good practices", "texttosearch", 1, "good practices", "texttosearch"),
+            ("good practices", "search", 1, "good practices", "texttosearch"),
+        ]
+    )
+    def test_api___simple_search_ok__by_content(
+        self, content_name, search_string, nb_content_result, first_content_name, content_body
```
Comment (lebouquetin) [inline backend/tracim_backend/tests/functional/test_simple_search.py]: same remark about naming and order of parameters

File: backend/tracim_backend/tests/functional/test_simple_search.py [hunk 4]
```diff
@@ -160,6 +160,135 @@ def test_api___simple_search_ok__by_filename(
...
+                "texttosearch",
+                1,
+                "good practices",
+                "texttosearch",
+                "another comment texttosearch",
+            ),
+        ]
+    )
+    def test_api___simple_search_ok__by_comment_content(
+        self,
```
Comment (lebouquetin) [inline backend/tracim_backend/tests/functional/test_simple_search.py]: same remark about naming and order

Comment (inkhey) [reply backend/tracim_backend/tests/functional/test_simple_search.py]: done

File: backend/tracim_backend/tests/functional/test_simple_search.py (hunk 3 above)
Comment (inkhey) [reply backend/tracim_backend/tests/functional/test_simple_search.py]: done.

File: backend/tracim_backend/tests/functional/test_elasticsearch_search.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/tests/functional/test_elasticsearch_search.py]: done.

File: backend/tracim_backend/tests/functional/test_elasticsearch_search.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/tests/functional/test_elasticsearch_search.py]: done.

**Pull request merged**